### PR TITLE
Fix position offset do not work when movingOnWindowChange is false

### LINKED ISSUE #63: position offset do not work when movingOnWindowChange is false

### PR DESCRIPTION
**Pull request by lzw-all-in**
The `position` parameter is not working when set `movingOnWindowChange  = false`.

Sometimes I want it to be a bit higher than the bottom center. And after I set the offset value just find it not work.
```dart
position: ToastPosition.bottom.copyWith(offset: -100)
```

Resolve #63 

|  Before   | After |
|  ----  | ----  |
| |   |

### CODE REVIEW

**Comment by lzw-all-in** [comment]
Hi @AlexV525 Could you help me to review this PR ? 

Thanks!

File: lib/src/widget/container.dart [hunk 1]
```diff
@@ -94,6 +90,14 @@ class _ToastContainerState extends State<ToastContainer>
     } else {
       offsetPadding = null;
     }
+
+    if (movingOnWindowChange != true) {
+      return Padding(
+        padding: offsetPadding ?? EdgeInsets.zero,
+        child: w,
+      );
+    }
```
Comment (AlexV525) [inline lib/src/widget/container.dart:99]: Can we continue to use the `AnimatedPadding` below instead of returning a new `Padding` widget?

Comment (lzw-all-in) [reply lib/src/widget/container.dart]: @AlexV525 Sorry for being late. It's reusing `AnimatedPadding` now. 

Thanks for your suggestion!

File: lib/src/widget/container.dart [hunk 2]
```diff
@@ -82,24 +82,24 @@ class _ToastContainerState extends State<ToastContainer>
...
       offsetPadding = EdgeInsets.only(top: offset);
-    } else if (offset < 0) {
+    } else {
       offsetPadding = EdgeInsets.only(bottom: offset.abs());
+    }
+
+    final EdgeInsets windowInsets;
+    if (movingOnWindowChange) {
+      windowInsets = EdgeInsets.only(
+        bottom: MediaQueryData.fromWindow(ui.window).viewInsets.bottom,
```
Comment (AlexV525) [inline lib/src/widget/container.dart:95]: Sorry for being late again (the notification was flushed in my queue). Can we get rid of .fromWindow? It's been deprecated IIRC.

Comment (lzw-all-in) [reply lib/src/widget/container.dart]: Hi @AlexV525, sorry for the late. Replace the deprecated method `fromWindow` to `fromView`. Could you help me review it again ? Thanks!

**Comment by AlexV525** [comment]
The PR looks good overall. Could you add some tests?

**Review by AlexV525 — APPROVED** [review]
Thanks for your contribution!

**Comment by lzw-all-in** [comment]
Hi @AlexV525 , I'm pushed a new change which fix the error in Github action.

**Pull request merged**